src/licensedcode/data/rules/nokia.yml

```diff
@@ -0,0 +1,4 @@
+license_expression: nokos-1.0a
+relevance: 76.22
```

Comment:
We usually use round numbers for relevance. Here I would use either 70 or 80
What has been your rationale to pick 76.22?

Reply:
Oh I thought about using the exact value just for the sake of precision. In case round numbers are preferred, I'll do the needful by setting the higher bound. Thanks for pointing it out!